From: Matthew Gamble <git@matthewgamble.net>

pyforge is incorrectly listed as a hard requirement in setup.py, when in fact
it's actually just for running the test suite for this library.

diff --git a/setup.py b/setup.py
index 9601262..62ec31f 100644
--- a/setup.py
+++ b/setup.py
@@ -25,7 +25,7 @@ setup(name="infi.unittest",
       version=__version__,
       packages=find_packages(exclude=["tests"]),
       namespace_packages=["infi"],
-      install_requires=_REQUIREMENTS,
+      install_requires=["munch"],
       scripts=[],
       entry_points = {
             'nose.plugins.0.10': [
